website_slides_qq_video/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Website Slides upload QQ video
+==============================
+This model  extends the functionality of and allows to publish the QQ video
+in the channel
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (website_slides)
+
+Bug Tracker
```

Comment:
@siyuancn I need a usage section explaining how to upload the qq videos and the format of the URL for qq.